client/my-sites/site-settings/theme-enhancements/index.jsx

```diff
@@ -103,7 +103,7 @@ class ThemeEnhancements extends Component {
 				<JetpackModuleToggle
 					siteId={ selectedSiteId }
 					moduleSlug="minileven"
-					label={ translate( 'Optimize your site with a mobile-friendly theme for tablets and phones.' ) }
+					label={ translate( 'Optimize your site with a mobile-friendly theme for tablets and phones' ) }
```

Comment:
:disappointed: 10 existing translations will be lost with this change.
If the string must change, the following option has already been translated **20** times:
`translate( 'Optimize your site with a mobile theme for tablets and phones.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).